src/features/expand-collapse-outdated-comments.js

```diff
@@ -0,0 +1,40 @@
+import select from 'select-dom';
+import {isMac, metaKey} from '../libs/utils';
+
+const keyName = isMac ? 'Alt' : 'Ctrl';
+
+function addTooltip(button) {
+	button.setAttribute('aria-label', keyName + ' + click to expand/collapse all outdated comments');
+	button.classList.add('tooltipped', 'tooltipped-n');
+}
+
+export default () => {
+	const showOutdatedButtons = select.all('.show-outdated-button, .hide-outdated-button');
+	showOutdatedButtons.forEach(button => {
+		addTooltip(button);
+		button.addEventListener('click', e => {
```

Comment:
I’d like this listener to be applied via jQuery delegates, so it works on all AJAX-loaded comments too.